tests/IResearch/CMakeLists.txt

```diff
@@ -79,7 +79,6 @@ add_executable(
   arangodbtests-iresearch
   ${ARANGODB_TESTS_COMMON_SOURCES}
   ${ARANGODB_IRESEARCH_TESTS_SOURCES}
-  ../main.cpp
```

Comment:
why main is removed?

Reply:
Got conflict during linkage. But maybe with latest devel merged it would change.